packages/kit/src/core/config/options.js

```diff
@@ -108,7 +109,11 @@ const options = object(
 			}),
 
 			csrf: object({
-				checkOrigin: boolean(true),
+				checkOrigin: deprecate(
+					boolean(true),
+					(keypath) =>
+						`\`${keypath}\` has been deprecated in favour of \`csrf.trustedOrigins: ['*']\`. It will be removed in a future version`
```

Comment:
`*`  is only correct for checkOrigin false, if someone had an explicit checkOrigin: true in theit config this message would be misleading

 what would be the correct value for "only allow deployed domain", empty array or not setting it?